diktat-rules/src/test/resources/test/paragraph2/kdoc/package/src/main/kotlin/org/cqfn/diktat/kdoc/methods/ThrowsTagInsertionExpected.kt

```diff
@@ -46,4 +46,22 @@ class Example {
         if (true) throw IllegalStateException("Lorem ipsum")
         else throw IllegalAccessException("Dolor sit amet")
     }
+
+    /**
+ * @throws RuntimeException
+ */
+fun test6() {
+        try {
+            foo()
+        } catch (_: NullPointerException) {
+            println("NPE!")
+        } catch (e: RuntimeException) {
+            println("Whoops...")
+            throw e
+        } catch (e: Error) {
+            val x = IllegalStateException()
```

Comment:
May be create issue on subject to improve `missingExceptions` detection to cover this case? When exception throws via local variable?

Or at least add `TODO` in code?

Reply:
https://github.com/cqfn/diKTat/issues/991